mne/gui/_coreg_gui.py

```diff
@@ -106,6 +106,7 @@ class CoregModel(HasPrivateTraits):
     has_fid_data = Property(Bool, depends_on=['mri_origin', 'hsp.nasion'],
                             desc="Required fiducials data is present.")
     has_pts_data = Property(Bool, depends_on=['mri.points', 'hsp.points'])
+    dig_points = Property(depends_on=['hsp.inst_points'])
```

Comment:
I don't think you're using this trait below, right? Maybe it's left over from an earlier implementation?